tasks/dogstatsd.py

```diff
@@ -171,19 +170,17 @@ def size_test(ctx, skip_build=False):
 
 
 @task
-def integration_tests(ctx, install_deps=False, race=False, remote_docker=False, go_mod="mod"):
+def integration_tests(ctx, race=False, remote_docker=False, go_mod="mod", timeout="10m"):
```

Comment:
What is the expected time for these tests? Are they supposed to last than 10 minutes ? If not I think we should modify default timeout so that we can run them without changing the default timeout

Reply:
The default timeout is 10m that's how I chose the default time WDYT?

Reply:
I think the default timeout should be a timeout that allows you to run the integration tests without being killed by `go test` timeout.

Reply:
Discussed offline: you're right that was off. Fixed with [bc30670](https://github.com/DataDog/datadog-agent/pull/29489/commits/bc306707fabdde1f660125262ca795019c743ee7) and [e45cd34](https://github.com/DataDog/datadog-agent/pull/29489/commits/e45cd341cc63a0bd66d298784f4ed8ef9a07730d).